opm/output/eclipse/summaryReader.cpp

```diff
@@ -244,7 +244,7 @@ void SummaryReader::findDeviations(std::vector<double>& absdev_vec,std::vector<d
       }
       else if((*referance_vec)[ivar]<(*checking_vec)[jvar]){
 	// Check with Linear polized arguments, jvar
-	double time_array[3];
+	double time_array[3]; // should contain { [time of occurance after referance] , [time of occurance before referance] , [time of referance] }
```

Comment:
> the `s/a/b` notation comes from sed and similar tools using regular expressions

Actually, it's older still.  The `s///` syntax for pattern matching and replacement originated in [ed](https://www.gnu.org/fun/jokes/ed-msg.html)--from which `sed` was created to be able to handle  files larger than could reasonably fit into memory and to automate common editing tasks.